DataConnectors/ZPA/Connector_LogAnalytics_agent_Zscaler_ZPA.json

```diff
@@ -0,0 +1,166 @@
+{
+    "id": "ZPA",
+    "title": "Zscaler Private Access",
+    "publisher": "Zscaler",
+    "descriptionMarkdown": "The [Zscaler Private Access (ZPA)](https://help.zscaler.com/zpa/what-zscaler-private-access) data connector provides the capability to ingest [Zscaler Private Access events](https://help.zscaler.com/zpa/log-streaming-service) into Azure Sentinel. Refer to [Zscaler Private Access documentation](https://help.zscaler.com/zpa) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **ZPAEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-ZPA-parser)",
+
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "ZPA_CL",
+            "baseQuery": "ZPA_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "All logs",
+            "query": "\nZPAEvent\n| sort by TimeGenerated"
```

Comment:
Invalid query, Can you please correct this? TimeGenerated is available in the Parser.

Reply:
added TimeGenerated to parser